examples/example_minimal.py

```diff
@@ -82,7 +80,8 @@ def objective(individual):
 
 ea_params = {"n_offsprings": 4, "tournament_size": 2, "n_processes": 2}
 
-evolve_params = {"max_generations": int(args["--max-generations"]), "min_fitness": 0.0}
+evolve_params = {"max_generations": np.inf, "min_fitness": 0.0, "max_objective_calls": 10000}
```

Comment:
is this change intentional or remaining from testing? if intentional, what's the motivation?

Reply:
It is still from testing. Should the new functionality be in a separate example (or just available and covered by tests)?

Reply:
i'd suggest to just have it available, documented and covered by tests. no need for an additional example.